### PR TITLE
Remove redhat-lsb removed from install_ceph_pre-requisities

### PR DESCRIPTION
**Pull request by veera-raghava-reddy**
This PR is to address the issue reported in https://github.com/red-hat-storage/cephci/issues/357 - stop installing redhat-lsb on RHEL

Removed rdhat-lsb from yum install cmd from 
conf/inventory/RHEL-8.0.0-x86_64-latest.yaml
conf/inventory/rhel-7.4-server-x86_64-released.yaml
conf/inventory/rhel-7.5-server-x86_64-latest.yaml
conf/inventory/rhel-7.5-server-x86_64-released.yaml
conf/inventory/rhel-7.6-server-x86_64-released.yaml
conf/inventory/rhel-7.7-server-x86_64.yaml
conf/inventory/rhel-7.8-server-x86_64.yaml
conf/inventory/rhel-7.9-server-x86_64.yaml
conf/inventory/rhel-8.1-server-x86_64.yaml
conf/inventory/rhel-8.2-server-x86_64.yaml
conf/inventory/rhel-8.3-server-x86_64.yaml
tests/misc_env/install_prereq.py

### CODE REVIEW

**Comment by veera-raghava-reddy** [comment]
Pre-requisite log before remove redhat-lsb - Total time ~7 mins
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1607660915153/install_ceph_pre-requisites_0.log

Pre-requisite log after removing redhat-lsb - Total ~5 mins
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1613584723122/install_ceph_pre-requisities_0.log

Test run after redhat-lsb - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1613584723122/result.html

**Comment by psathyan** [comment]
@veera-raghava-reddy I am not sure if we should pursue this PR. At present, none of the packages are being installed as the is no repo configured in the VM. Hence, none of the packages are installed.

**Comment by veera-raghava-reddy** [comment]
In the test log prior to remvoing redhat-lsb, there are entries of installing packages -
redhat-lsb-core                        x86_64  4.1-47.el8                              rhel-8-for-x86_64-appstream-rpms   45 k
 redhat-lsb-cxx                         x86_64  4.1-47.el8                              rhel-8-for-x86_64-appstream-rpms   23 k
 redhat-lsb-desktop                     x86_64  4.1-47.el8                      
2020-12-11 04:33:58,174 - ceph.ceph - INFO -         rhel-8-for-x86_64-appstream-rpms   26 k
 redhat-lsb-languages                   x86_64  4.1-47.el8                              rhel-8-for-x86_64-appstream-rpms   25 k
 redhat-lsb-printing                    x86_64  4.1-47.el8                              rhel-8-for-x86_64-appstream-rpms   23 k
 redhat-lsb-submod-multimedia           x86_64  4.1-47.el8                              rhel-8-for-x86_64-appstream-rpms   22 k
 redhat-lsb-submod-security             x86_64  4.1-47.el8                              rhel-8-for-x86_64-appstream-rpms   22 k

**Comment by psathyan** [comment]
Sorry, I didn't notice the below [change](https://github.com/red-hat-storage/cephci/pull/370/files#diff-a2a99cf39f88141c6551d674cc418fa009455f8f3717b41f5a126e80f1097515R30).

I believe this is the only change that is required.

**Comment by veera-raghava-reddy** [comment]
yes, remove redhat-lsb

File: tests/misc_env/install_prereq.py [hunk 1]
```diff
@@ -24,12 +23,11 @@
         "wget",
         "git",
         "python3-virtualenv",
-        "redhat-lsb",
         "python3-nose",
         "python3-pip",
     ],
 }
-deb_packages = ["wget", "git", "python-virtualenv", "lsb-release", "ntp"]
```
Comment (ktdreyer) [inline tests/misc_env/install_prereq.py:32]: We *do* need to keep this one for Ubuntu, because we use the `lsb_release` command.

Comment (veera-raghava-reddy) [reply tests/misc_env/install_prereq.py:32]: Noted. Will change it.

**Comment by vasukulkarni** [comment]
@veera-raghava-reddy your commit has merges - 662e663f055104624fb9cbfeeb8279bae26be376 , you should rebase it instead.

**Review by rakeshgm — APPROVED** [review]

**Comment by ktdreyer** [comment]
Thanks! And if there are workunits that use `lsb_release` or anything else in `redhat-lsb`, let me know, and we can rewrite those to use `/etc/os-release` or something more lighter-weight.

**Pull request merged**